devfile.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: 1.0.0
+metadata:
+  generateName: rh-che-
+attributes:
+  persistVolumes: 'false'
+projects:
```

Comment:
FYI you can omit current repository now

Reply:
@benoitf is it available in 7.8.0?

Reply:
yes https://github.com/eclipse/che/commit/dc2f3b5ea9e1ce289ff13e43d46734835423cdf5

Reply:
![image](https://user-images.githubusercontent.com/436777/74040389-ed7c0600-49c3-11ea-8e07-8c806275831a.png)

Reply:
@benoitf @mshaposhnik but looks like this will not work for things like `https://che.openshift.io/f?url=https://raw.githubusercontent.com/redhat-developer/rh-che/master/devfile.yaml` what url format is expected to be used?

Reply:
```
https://che.openshift.io/f?url=https://github.com/redhat-developer/rh-che
https://che.openshift.io/f?url=https://github.com/redhat-developer/rh-che/tree/master

for `branch-name`
```
https://che.openshift.io/f?url=https://github.com/redhat-developer/rh-che/tree/<branch-name>
```

Reply:
Fixed in https://github.com/redhat-developer/rh-che/pull/1796